oracle-e2e/envs/contracts-deploy.env

```diff
@@ -4,6 +4,8 @@ DEPLOYMENT_GAS_LIMIT=4600000
 HOME_DEPLOYMENT_GAS_PRICE=10000000000
 FOREIGN_DEPLOYMENT_GAS_PRICE=10000000000
 GET_RECEIPT_INTERVAL_IN_MILLISECONDS=50
+DEPLOYMENT_GAS_LIMIT_EXTRA=0.2
+DEPLOY_REWARDABLE_TOKEN=false
```

Comment:
`DEPLOYMENT_GAS_LIMIT` was replaced by `DEPLOYMENT_GAS_LIMIT_EXTRA` so it is not needed any more.

Please correct this in other `.env` files.

Reply:
Done.